server.js

```diff
@@ -89,7 +102,7 @@ app.get("/countries/:country", async function (req, res) {
     return;
   }
   // adding status code 404 not found and sending response
-  res.status(404).send({ message: "Country not found or Has no cases" });
+  res.status(404).send({ message: "Country not found or doesn't have cases" });
```

Comment:
doesn't have **any** cases maybe? this doesn't sound very fluent

Reply:
sorry fixed